archive/Vagrantfile

```diff
@@ -34,10 +34,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "chef/centos-6.5"
```

Comment:
Maybe is better to have an archive/vagrant instead that using the archive/ "root".

In addition, move the doc/manuals/vagrant.md to the archive (removing the CoAP reference in that file) and the link to it in the master README.md.